x-pack/plugins/secops/public/components/page/navigation/index.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+
+import {
+  EuiButton,
+  EuiSpacer,
+  // @ts-ignore
+  EuiTab,
+  // @ts-ignore
+  EuiTabs,
+} from '@elastic/eui';
+import styled from 'styled-components';
+
+import { getHostsUrl, getNetworkUrl, getOverviewUrl } from '../../link_to';
+
+interface NavTab {
+  id: string;
+  name: string;
+  href: string;
+  disabled: boolean;
+}
+
+const navTabs: NavTab[] = [
+  {
+    id: 'overview',
+    name: 'Overview',
+    href: getOverviewUrl(),
+    disabled: false,
+  },
+  {
+    id: 'hosts',
+    name: 'Hosts',
+    href: getHostsUrl(),
+    disabled: false,
+  },
+  {
+    id: 'network',
+    name: 'Network',
+    href: getNetworkUrl(),
+    disabled: false,
+  },
+];
+
+export const Navigation: React.SFC = () => (
+  <div>
+    <EuiTabs>{renderTabs()}</EuiTabs>
+    <AddData />
+    <EuiSpacer />
+  </div>
+);
+
+const renderTabs = () =>
+  navTabs.map((tab: NavTab) => (
+    <EuiTab
+      onClick={() => window.location.assign(tab.href)}
```

Comment:
not requesting a change, just noting a general preference for not using lambdas in click handlers (to prevent re-renders)

Reply:
👍